test/unit/meta/traits/common_logical.cpp

```diff
@@ -0,0 +1,79 @@
+//==================================================================================================
+/**
+  EVE - Expressive Vector Engine
+  Copyright : EVE Project Contributors
+  SPDX-License-Identifier: BSL-1.0
+**/
+//==================================================================================================
+
+#include "test.hpp"
+#include "unit/module/core/logical_test.hpp"
+
+#include <eve/module/core.hpp>
+
+TTS_CASE("eve::common_logical on boolean x boolean")
+{
+  TTS_TYPE_IS((eve::common_logical_t<bool, bool>), bool);
+};
```

Comment:
Nit: 

common_logical_t is always simmetrical. Extract a function:

```
template <typename L, typename R, typename Res>
void common_logical_res_test()
```

And use that one. This way you halve the number of test cases you need to write. 

You can also put all "logical/non-logical" common logical stuff in there.